docs/01-app/03-api-reference/04-functions/use-link-status.mdx

```diff
@@ -98,9 +100,9 @@ const { pending } = useLinkStatus()
 
 ## Example
 
-### Inline loading indicator
+### Inline link hint
 
-It's helpful to add visual feedback that navigation is happening in case the user clicks a link before prefetching is complete.
+Add a subtle visual hint next to link text to confirm a click when prefetching hasn’t completed.
```

Comment:
I don't think we should suggest adding the hint "next to link text". Ideally it's a visual change that can't cause any layout shifts